ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/customercenter/viewmodel/CustomerCenterViewModel.kt

```diff
@@ -281,13 +296,34 @@ internal class CustomerCenterViewModelImpl(
         purchaseInformation: PurchaseInformation?,
         screen: CustomerCenterConfigData.Screen,
     ): List<CustomerCenterConfigData.HelpPath> {
-        return purchaseInformation?.let { info ->
-            if (info.isLifetime) {
-                screen.supportedPaths.filter { it.type != CustomerCenterConfigData.HelpPath.PathType.CANCEL }
-            } else {
-                screen.supportedPaths
-            }
-        } ?: emptyList()
+        return screen.paths
+            .filter { isPathAllowedForStore(it, purchaseInformation) }
+            .filter { isPathAllowedForLifetimeSubscription(it, purchaseInformation) }
+    }
+
+    private fun isPathAllowedForLifetimeSubscription(
+        path: CustomerCenterConfigData.HelpPath,
+        purchaseInformation: PurchaseInformation?,
+    ): Boolean {
+        if (purchaseInformation?.isLifetime != true) return true
+        return path.type != CustomerCenterConfigData.HelpPath.PathType.CANCEL
+    }
+
+    private fun isPathAllowedForStore(
+        path: CustomerCenterConfigData.HelpPath,
+        purchaseInformation: PurchaseInformation?,
+    ): Boolean {
+        return when (path.type) {
+            HelpPath.PathType.MISSING_PURCHASE,
+            HelpPath.PathType.CUSTOM_URL,
+            -> true
+            HelpPath.PathType.CANCEL ->
+                purchaseInformation?.store == Store.PLAY_STORE || purchaseInformation?.managementURL != null
+            HelpPath.PathType.REFUND_REQUEST,
+            HelpPath.PathType.CHANGE_PLANS,
+            HelpPath.PathType.UNKNOWN,
+            -> false
+        }
     }
```

Comment:
Don't we need to add the store check for play store?

Reply:
talked offline but not needed, it's just not available for any platform